content/docs/dvclive/api-reference/live/log_metric.md

```diff
@@ -0,0 +1,80 @@
+# Live.log_metric()
+
+```py
+ def log_metric(name: str, val: float):
+```
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
+
+live.log_metric("train/loss", 0.4)
+live.log_metric("val/loss", 0.9)
+```
+
+## Description
+
+On each `live.log_metric(name, val)` call DVCLive will create a _metrics
+history_ file in `{Live.plots_dir}/metrics/{name}.tsv`:
+
+```
+├── dvclive
+│   └── plots
+│       └── metrics
+│           ├── train
+│           │   └── loss.tsv
+│           └── val
+│               └── loss.tsv
+└── dvclive.json
```

Comment:
**Blocking**: this looks like it's based on the old output format. Also, given the example above, there shouldn't be any summary file, right?